### DISCUSSION TITLE
Reverting How You Upload Images

### DISCUSSION THREAD

**Discussion by LivingNextDoorToAlice** [MKT-Playbook/Hub#41] [Feature requests]
As of this new update - which has been really great - it seems that the app now only accepts screenshots. While I know why this is, I really did like the freedom of being able to choose any photo - I happened to like customizing my Playbook similar to a Pokedex. Is it possible to revert this change so uploading an image is just how it was before the update? Everything else is great otherwise! 😁

**Comment by TitusPosiadly** [comment]
Hey, thanks for the feedback.
You are totally right. I forgot to reenable all images for the photo picker when importing just a single image (for the card detail page or the manual add page). I am guessing you are using those to import custom images or are you using the bulk import with the scanner?
